noxfile.py

```diff
@@ -348,6 +348,12 @@ def graphql_codegen_schema_change(session: nox.Session) -> None:
     )
 
 
+@nox.session(python="3.10", name="graphql-codegen-python")
+def graphql_codegen_python(session: nox.Session) -> None:
+    """Runs GraphQL codegen to generate python code."""
+    session.run("./tools/pydantic_codegen/generate-graphql-python.sh")
+
+
```

Comment:
**note:** not sure if it'd be better to absorb this into the existing `graphql-codegen-schema-change` nox session (for golang codegen).

Open to any preferred alternatives, though.